src/store.c

```diff
@@ -351,14 +352,18 @@ int pthreads_store_write(zval *object, zval *key, zval *write) {
 				normal refcounting, we'll just never read the reference
 			*/
 			rebuild_object_properties(&threaded->std);
+			
+			pthreads_store_sync(object);
```

Comment:
This has the potential to make unsynchronized writes extremely slow ... I don't like it ... would be good to find another way to do whatever you are trying to do ...